docs/network.md

```diff
@@ -0,0 +1,55 @@
+# 8 Routing table computation
+
+Routing table computation does a few things:
+- for each peer `B`, calculates set of peers `|C_b|`, such that each peer is on the shortest path to `B`.
```

Comment:
```suggestion
- for each peer `B`, calculates set of peers `|C_b|`, such that each peer is on a shortest path to `B`.
```

Reply:
Also worth documenting how "distance" is measured when talking about shortest paths.

Reply:
@bowenwang1996 Thanks, I added the following
```
- The distance is calculated as the minimum number of nodes on the path from given node `A`, to each other node 
on the network. That is, `A` has a distance of `0` to itself. It's neighbors will have a distance of `1`.
The neighbors of theirs neighbors will have a distance of `2`, etc.
```